.github/docs/contributing.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to aoi.js ❤
+
+When contributing to this repository, please first discuss the change you wish to make via issue, Discord or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process 🧩
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+
+2. Update the SUMMARY.md with details of changes to the interface, this includes new useful file locations and container parameters.
+
+3. Increase the version numbers in any examples files and the README.md to the new version of **aoi.js**. Pull Request would represent.
+
+4. You may merge the Pull Request in once you have the sign-off of the developers, or if you do not have permission to do that, you may request the reviewer to merge it for you.
+
+## ✦ Code of Conduct ✦
```

Comment:
This can be separate markdown.

Reply:
I'll be seperating CODE OF CONDUCT then.